ginga/util/ap_region.py

```diff
@@ -0,0 +1,285 @@
+#
+# ap_region.py -- AstroPy regions support
+#
+# This is open-source software licensed under a BSD license.
+# Please see the file LICENSE.txt for details.
+#
+"""
+This module provides Ginga support for ds9 type region files and objects via
+the astropy regions module.
+"""
+import numpy as np
+
+from ginga.canvas.CanvasObject import get_canvas_types
+
+import regions
```

Comment:
@ejeschke , I think this import needs to be inside a `try ... except ...` block because it is *optional* dependency.

Reply:
Hmm, are you still sure about this one, @pllim?  Right now there is nothing in Ginga that imports `ap_region`.  I think until there is another module (e.g. a plugin) that imports it, it is not necessary to wrap it in a `try` clause.  At such a time it is necessary we can add it.

Currently if you want to use this module you *have* to have `regions` installed, and I think the traceback from the import error tells you everything you need to know if you try to use it without `regions`.

Reply:
Hmm, except possibly the auto api sphinx builder stuff?

Reply:
Yes, RTD will fail unless you add `regions` as a doc build dependency.

Reply:
Ok, I added a try/except around it.